### PR TITLE
Updated section S3-compatible Object Storage in docs "storage-uri.md"

### PR DESCRIPTION
**Pull request by maximkpa**
### Description

The methods used were proposed by Saroh in a closed issue #1584 

Added note with the endpoint url for Google Cloud Storage
Added instructions to get and to run commands for AWS_SECRET_ACCESS_KEY & AWS_ACCESS_KEY_ID
Typo fixes

### How was this PR tested?

Tried to create an index with Google Cloud Storage. Only works if access token, secret token, and the right GCS endpoint url assigned.

### CODE REVIEW

File: docs/reference/storage-uri.md [hunk 1]
```diff
@@ -69,15 +69,26 @@ The region or custom endpoint will be detected using the first successful method
...
 
 
-Quickwit can target other S3-compatible storage like MinIO.
+Quickwit can target other S3-compatible storage.
 This is done by setting an endpoint url in the `QW_S3_ENDPOINT` environment variable.
 
 In this case, the region will be ignored.
 
-For instance:
+Set up the endpoint url:
```
Comment (fmassot) [inline docs/reference/storage-uri.md:80]: We are using `URI` everywhere in the code base, it would be better to use it here

Comment (fmassot) [inline docs/reference/storage-uri.md:80]:
```suggestion
Example of using Google Cloud Storage: 
```bash
export QW_S3_ENDPOINT=https://storage.googleapis.com
```
```

Comment (maximkpa) [reply docs/reference/storage-uri.md]: Put "Example" and "Example for Google Cloud Storage" instead of a "Note" after, looks very clean

**Comment by fmassot** [comment]
@maximkpa can you close this PR and update the storage URI doc on your PR #1782 ? You will be able to put a link to the setup guides.

**Comment by maximkpa** [comment]
Closed (Adding Info to #1782)

**Pull request closed without merging**